incognia/api.py

```diff
@@ -34,7 +37,10 @@ def register_new_signup(self,
                 'installation_id': installation_id,
                 'address_line': address_line,
                 'structured_address': structured_address,
-                'address_coordinates': address_coordinates
+                'address_coordinates': address_coordinates,
+                'external_id': external_id,
+                'policy_id': policy_id,
+                'account_id': account_id
```

Comment:
What about writing a test, wdyt?

Reply:
added a test, with the address information also ;)